draft-ietf-quic-recovery.md

```diff
@@ -1517,7 +1494,7 @@ Invoked from DetectLostPackets when packets are deemed lost.
      CongestionEvent(largest_lost_packet.time_sent)
```

Comment:
How about also changing this to the following, and removing the line before this (last() is a language-specific method, so I'm suggesting largest())?
```suggestion
     CongestionEvent(lost_packets.largest().time_sent)
```

Reply:
I'm not a big fan of largest here and above.  If you want to make the change, maybe another PR, since this one is already quite large.